courses/free-courses-hi.md

```diff
@@ -99,6 +118,13 @@
 * [Web Development Tutorials for Beginners in Hindi: HTML, CSS, JavaScript and more](https://www.youtube.com/playlist?list=PLu0W_9lII9agiCUZYRsvtGTXdxkzPyItg) - CodeWithHarry
 
 
+#### Bootstrap
+
+* [Bootstrap 3 Tutorial](https://youtube.com/playlist?list=PLjVLYmrlmjGciJ5_Ze6jDIfJpEeMYtsSe) - WsCube Tech
+* [Bootstrap Tutorial In Hindi](https://youtube.com/playlist?list=PLdPwRNmUlk0k91-qAXTHFqMScNEuo8E5d) - CODE4EDUCATION
```

Comment:
Patch to recover 3 diffs with #7005 committed at 9fadbf98af17205729f3e562a4fcbd610b83bdf0 deleted at 8b60e6f190880a378a43ad54b0ef3b3ec43f550e but alphabetizing this section: 
https://github.com/EbookFoundation/free-programming-books/pull/7005/commits/9fadbf98af17205729f3e562a4fcbd610b83bdf0#diff-0bed9be5a9c29db386438a12b2e1abaac6945680b2e2ab9d07c0b58063e315adR110-R114

```suggestion
* [Bootstrap Tutorial for Beginners(Hindi)](https://youtube.com/playlist?list=PLgPJX9sVy92wc38jA6JtvkA4l1xmJcKKH) - CS Geeks
* [Bootstrap Tutorial In Hindi](https://youtube.com/playlist?list=PLdPwRNmUlk0k91-qAXTHFqMScNEuo8E5d) - CODE4EDUCATION
* [Bootstrap Tutorial In Hindi](https://youtu.be/vpAJ0s5S2t0) - CodeWithHarry
* [Bootstrap Tutorial in Hindi With 2 Projects for Beginners \| Complete Bootstrap 5 Tutorial in Hindi](https://youtu.be/QE5oQh63gGE) - Tech Gun
```